### ISSUE TITLE
Can't see all search results in model picker

### ISSUE THREAD

**Issue by kamilmielnik** [metabase/metabase#32748]
### Describe the bug

https://github.com/metabase/metabase/assets/6830683/884fb8e6-94db-419f-9b7a-072cbe6736c4

### To Reproduce

1. Create lots of models (I had 14)
2. Create new action (with + New button)
3. Use sample database and any query, e.g. `update reviews set rating = 5 where 1 = 2;`
4. Click "Save", modal will open
5. Click "Select a model", dropdown will open

I can't see all options in the dropdown.
I can't scroll the dropdown to see the last option.

It gets worse as height of the viewport decreases.

### Expected behavior

You can see all options in the dropdown or the dropdown is scrollable so that you can scroll to the last option.
### Information about your Metabase installation

```JSON
You're on version v0.47.1-SNAPSHOT
Built on 2023-07-28

Src_hash: d2a447f20d7b7cbd4a7db387df96702e81fd0ced

Branch: 32350-actions-tests

Hash: f3094f6
```

### Severity

P3

### Additional context

It can be worked around with search as long as model names are unique.

**Comment by iethree** [comment]
this will be handled by https://github.com/metabase/metabase/issues/36376

**Comment by iethree** [comment]
closed by #36376
